server/api/student/student.controller.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var _ = require('lodash');
+var Student = require('./student.model');
```

Comment:
With the change to the exports I believe this needs to be:

```
var Student = require('./student.model').Student;
```

This is causing the TravisCI test to fail and the routes to return status 500.